test/jdk/java/nio/channels/DatagramChannel/StressNativeSignal.java

```diff
@@ -74,20 +70,37 @@ public void shutdown() {
         }
     }
 
+    public void waitForTestThreadsToStart() {
+
+        udpThread.waitTestThreadStart();
+        serverSocketThread.waitTestThreadStart();
+    }
+
     public class ServerSocketThread extends Thread {
         private volatile boolean shouldTerminate;
         private ServerSocket socket;
+        private volatile CountDownLatch threadStarted = new CountDownLatch(1);
```

Comment:
This should be final rather than volatile.

Reply:
done